urpautils/universal.py

```diff
@@ -362,3 +362,22 @@ def get_previous_work_day_date(today: datetime.date = datetime.date.today(), cou
         if not previous_day in holiday_days and not previous_day.weekday() in (5, 6):
             return previous_day
     raise RuntimeError("No previous working day found")
+
+
+def robot_has_time(start: str = "00:00:00", end: str = "23:59:59") -> bool:
+    """Returns True if current time is in range start <= current time <= end, else False
+
+    :param start:               str, in format HH:MM:SS, by default "00:00:00"
+    :param end:                 str, in format HH:MM:SS, by default "23:59:59"
+    :return                     bool
+    """
+    now = datetime.datetime.now()
+    try:
```

Comment:
This will work fine in one day. But what if robot starts at 22:00:00 and ends at 02:00:00.
There must be some kind of functionality to handle this. Pseudocode example:

```
if start > end:
    start_time = ... .replace(year=yesterday.year, month=yesterday.month, day=yesterday.day)
```